octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,13 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        System.out.println("Sum of firstNumber and secondNumber is: " + add(10, 20));
+    }
+
+    private static int add(int firstNumber, int secondNumber) {
+        return firstNumber + secondNumber;
```

Comment:
[Severity: Minor] The return type of the 'add' method is 'int', which is fine for this case, but consider what happens if you want to handle larger numbers or decimal values in the future.
You might want to use 'double' for more flexibility with number types.
Rationale: Using 'double' allows for a wider range of numeric inputs, including fractions.
Reference: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html